scripts/installGUI

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+export WITH_MESH_POINT=false
+export WITH_WIFI_AP=false
+export WITH_IPFS=false
+export WITH_PROMETHEUS_NODE_EXPORTER=false
+export WITH_PROMETHEUS_SERVER=false
+export WITH_GRAFANA=false
+export WITH_EXTRA_TOOLS=false
+export WITH_H_DNS=false
+export WITH_H_NTP=false
+export WITH_FAKE_HWCLOCK=false
+
+export SET_MESH_POINT=off
+export SET_WIFI_AP=off
+export SET_IPFS=off
+export SET_PROMETHEUS_NODE_EXPORTER=off
+export SET_PROMETHEUS_SERVER=off
+export SET_GRAFANA=off
+export SET_EXTRA_TOOLS=off
+export SET_H_DNS=off
+export SET_H_NTP=off
+export SET_FAKE_HWCLOCK=off
+
+sudo apt-get install dialog
+
+dialog --backtitle "TOMesh Prototype Node" --title "Select feature starting point (Customize next)" --menu "Presets" 20 70 10 \
+        BASIC "Basic" \
```

Comment:
Are we generally using `Basic` to describe the Mesh Point w/ AP? Seems ambiguous in this context.

Reply:
This was just a quick slap together list as a possible idea to extend @benhylau of setting  everything off, this way there are PRESETS.

We can create whatever groups we want i guess that needs to be decided. Its just a starting point:)

AP is a weird thing because in current state it ONLY works with the onboard wifi on a rpi3 and doesn't actually ask you if you want to install if unless your running an RPI3

MeshScripts PR will fix this